### PR TITLE
Colorway: add dsa milkshake

### PR DESCRIPTION
**Pull request by zekth**
With approval from Biip
Screenshot 2019-08-19 at 12 05 15

### CODE REVIEW

**Comment by zekth** [comment]
@yanfali its really weird because i just use 'yarn lint'

**Comment by noroadsleft** [comment]
Differing lint settings, perhaps? I always do my linting by hand. :man_shrugging: 

This is more a stylistic complaint, but I don't like the way pure white keys look in light mode. Something like `background: #fdfbf7` for `.dsa-milkshake-key` and `.dsa-milkshake-mod` might look a bit nicer (though I'm not sure how true-to-life that would be as I don't own this set). Just something a bit off-white to bring out the keys against the light mode background.

Also I realized while testing this that there's a bug of sorts with key legends in dark mode versus light mode. Should be fixed, but out-of-scope here.

**Comment by zekth** [comment]
I've asked biip and the white is just pure white #FFF so...

About linting it uses the lint config defined in the project because i only use `yarn lint`. But on this project it's totally weird and i don't really like the abstract way of the vue-clie to use prettier. I'd better use raw prettier command.

**Comment by noroadsleft** [comment]
K, maybe a different color for `#visual-keymap` would help overall, which would be for another PR.

I'll defer to @yanfali as for the lint/prettier implementation.

File: src/components/colorways/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import dsa_galaxy_class from './dsa/galaxy_class';
+import milkshake from './dsa/milkshake';
```
Comment (yanfali) [inline src/components/colorways/index.js]: use the same naming convention. There's going to be a Kat Milkshake soon.

**Pull request merged**